api/src/main/java/com/okta/authn/sdk/resource/VerifyPassCodeFactorRequest.java

```diff
@@ -21,7 +21,19 @@
 
     VerifyPassCodeFactorRequest setPassCode(String passCode);
 
-    Boolean  getRememberDevice();
+    /**
+     * @deprecated 'rememberDevice' property is no longer a body param but an url/query param.
```

Comment:
nit: change "no longer" to "not"

```suggestion
     * @deprecated 'rememberDevice' property is not a body param but an url/query param.
```